src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1030,9 +1029,10 @@ public void testClearLazyLoadCacheInDetailAST() throws Exception {
         final DefaultConfiguration checkerConfig = createRootConfig(treeWalkerConfig);
 
         final String filePath = getPath("InputCheckerClearDetailAstLazyLoadCache.java");
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
 
-        verify(checkerConfig, filePath, expected);
+        execute(checkerConfig, filePath);
+
+        assertWithMessage("exception expected");
```

Comment:
exception is not expected, please look around we do have examples of few modules in config comment in Input files, see inputs for filters.